lib/std/std.zig

```diff
@@ -191,7 +204,7 @@ pub const options = struct {
     else
         debug.default_enable_segfault_handler;
 
-    /// Function used to implement std.fs.cwd for wasi.
+    /// Function used to implement std.fs.cwd for WASI.
```

Comment:
```suggestion
    /// Function used to implement `std.fs.cwd` for WASI.
```

(rendered as code by autodoc, possibly linkified)